controllers/links.js

```diff
@@ -0,0 +1,38 @@
+var express = require('express');
+var router = express.Router();
+var db = require('../models');
+var ejsLayouts = require('express-ejs-layouts');
+var Hashids = require("hashids");
+var hashids = new Hashids("da39a3ee5e6b4b0d3255bfef95601890afd80709")
+
+router.post('/', function(req, res) {
+  db.link.findOrCreate({
+    where: { url: req.body.linky }
+  }).spread(function(data, create) {
+    var hash = hashids.encode(data.id);
+
+    if (create) {
+      // create object if not in database
+      res.send("Not in database: " + process.env.LINK_SHORTENER_URL + '/' + hash)
```

Comment:
process.env.LINK_SHORTENER_URL - what API link are you using? I don't have it as it's part of the git ignore so it doesn't work for me. I replaced it with localhost:3000 and the links worked. I'm just curious which API you are using.